### ISSUE TITLE
Fold the brave-keyring Debian/RPM package into brave-browser to simplify Linux installation

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#9050]
Both Chrome and Vivaldi offer Debian/RPM packages to download from their websites. Once installed, they automatically add a new GPG key to the trusted keyring and then enable automatic updates.

We deviated from the packaging that's built into Chromium in order to modernize it and facilitate key rotation. This issue is about revisiting this decision based on our experience rotating signing keys and extending expiry dates.

In other words, we would offer a single `.deb`/`.rpm` that would:
- install the browser
- install our signing key to the local keyring
- install our repository

Originally suggested on https://twitter.com/PavanGayakwad/status/1246673744900182018.

**Comment by fmarier** [comment]
When/if we do this, we'll have to be careful because of potential conflicts with `extrepo`: https://microsoftteams.uservoice.com/forums/908686-bug-reports/suggestions/42938283-on-debian-installation-through-the-extrepo-util

**Comment by NayamAmarshe** [comment]
Any updates on this? It's been more than a year.

**Comment by freechelmi** [comment]
Thanks for this issue , My LINUX clients were really upset that downloading brave brought them on this cryptic page  https://brave.com/linux/

if we want LINUX to cross 5% for 2025 we need to treat linux users as dumb as possible.

**Comment by pavangayakwad** [comment]
As the time passes, Brave .deb and .rpm is needed more than often for easy download and install for non-techi users. Hope you guys prioritize it.
